### ISSUE TITLE
Customizer functionality is broken

### ISSUE THREAD

**Issue by christophherr** [Getbeans/Beans#260]
The Preview Tools stopped working with WordPress 4.7 and up as the slider does not change width or height of the previewer's iframe.

**Comment by hellofromtonya** [comment]
This bug has multiple problems.  We should note that the problems began with WordPress 4.7 and up, as Customizer changed.

Problems:

1. As of WP 4.7, Customizer changed how we bind to each field to trigger work in the previewer.
2. Our customer slider's `.keyup()` event on change is not being picked up Customizer.
3. CSS issues between WP 4.2 and down.

## Customizer binding change

Previously, we could use `wp.customize.value()` and then bind to it for event triggers.  As of WP 4.7, the new Customizer API changes that to be:

```
var that = this;

wp.customize( 'beans_viewport_width', function( setting ) {
    setting.bind( that.viewportWidth.bind(that) );
});
```

Upon fixing all the other issues, the old way still works, but does not appear to be the preferred way to bind to Customizer.

## Customizer not picking up on the `keyup` event.

I'm guessing it's bound to the `change` event instead. 

Fix: Switch to `.trigger( 'change' );`. Bam it works.

**Comment by hellofromtonya** [comment]
>As of WP 4.7, the new Customizer API changes that to be:

I misspoke.  The binding methodology that Weston has suggested was available in WP 4.2.  Aha! 

Fix: Let's use his preferred approach.
